content/modules/search/pages/config-options.adoc

```diff
@@ -77,4 +79,33 @@ jmix.search.changedEntitiesIndexingEnabled = false
 To utilize the functionality of the Search add-on, users with limited access to the system should have one of the following roles:
 
 * *Search: edit filter* provides permissions to add full-text search conditions to the filter.
-* *Search: view search results* provides permissions to access the Search Results view.
\ No newline at end of file
+* *Search: view search results* provides permissions to access the Search Results view.
+
+[[index-settings]]
+== Index Settings and Analysis
+Add-on allows to configure index settings including analysis.
+
+NOTE: This configuration depends on used search platform (OpenSearch or Elasticsearch) and should be recreated accordingly if you switch between them.
+
+* Create spring bean that implements `OpenSearchIndexSettingsConfigurer` (or `ElasticsearchIndexSettingsConfigurer` in case of Elasticsearch usage).
+* Implement method `configure(OpenSearchIndexSettingsConfigurationContext)`.
+
+=== Implementation
+
+Settings can be configured via settings builder (belongs to OpenSearch\Elasticsearch Java API client).
```

Comment:
Use forward slash here and in other places: OpenSearch/Elasticsearch

Reply:
Done